web/app/templates/tournament_form.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+
+
+
+{% block content %}
+<div class="container">
+
+<h2>Please enter Tournament Info:</h2>
+  <form method="post" name='new'>
+    {{ form.hidden_tag() }}
+
+        <p>Event Name: {{ form.event_name() }}</p></br>
+
+        <p>Venue: {{ form.venue() }}</p></br>
+
+        <p>Director: {{ form.director }}</p></br>
+
+        <p>Pairing: {{ form.pairing }}</p></br>
+
+        <p>Rule Set: {{ form.rule_set }}</p></br>
+
+        <p>Check to Submit Finalized Tournament Record:  {{ form.submitted }}</p></br>
```

Comment:
submitting will need to be a separate form. but fine for now -- we can update later.